npc/merchants/advanced_refiner.txt

```diff
@@ -123,7 +123,7 @@ S_RefineValidate:
 	mes "Would you like to continue?";
 	next;
 	if(select("Yes", "No") == 1) {
-		if (getequippercentrefinery(getarg(3)) < 100) {
+		if (getequippercentrefinery(getarg(3), true) < 100) {
```

Comment:
hmm ... `true`?? aren't suppose to use the documented constant here? like `REFINE_CHANCE_TYPE_XXXXX` ?